src/exportStrategies.js

```diff
@@ -37,64 +37,49 @@ export const stream = _.curry(async ({ strategy, stream }) => {
   stream.end()
 })
 
-// Strategies with a custom formatter
-
-// This format function breaks down an array of plain objects into key value pairs,
-// which will be formatted using the given format rules,
-// which then will return the array of formatted plain objects.
-// If the format rules doesn't have a proper way to handle certain key/value pair,
-// it will resort to the defaultLabel and defaultDisplay parameters, which by default
-// change the key to be in startCase form (so "ABCDary" becomes "ABC Dary"), and keeps the value as is.
-const format = (
-  rules,
-  defaultLabel = _.startCase,
+// Format object values based on passed formatter or _.identity
+// Also fill in any keys which are present in the included keys but not in the passed in object
+export const formatValues = (
+  rules = {},
+  includedKeys = [],
   defaultDisplay = _.identity
-) => {
-  let propertyFormatter = _.map(([k, v]) =>
-    rules[k]
-      ? [
-          rules[k].label || defaultLabel(k),
-          (rules[k].display || defaultDisplay)(v),
-        ]
-      : [defaultLabel(k), defaultDisplay(v)]
-  )
-  return _.map(
-    _.flow(
-      F.flattenObject,
-      // Extending each element with properties that might be missing
-      x => ({
-        ..._.mapValues(() => '', rules),
-        ...x,
-      }),
-      _.toPairs,
-      propertyFormatter,
-      _.fromPairs
+) =>
+  _.map(obj => {
+    // Format all values of the passed in object
+    let resultObject = F.mapValuesIndexed(
+      (value, key) =>
+        rules[key]
+          ? (rules[key].display || defaultDisplay)(value)
+          : defaultDisplay(value),
+      obj
     )
+    // Fill the empty properties for the objects missing the expected keys
```

Comment:
If there's an explicit set of `includedKeys`, they'd be on the `includes` of the results, right? I think `includedKeys` will always be present on the results so we can just remove this.

Reply:
Well no this is why the whole problem started in the first place. The ES result coming in has 3 properties of the 6 expected. The remaining ones are not part of the object at all. This is why the export only showed 3 columns and skipped the rest since it was taking the headers from the first row.